src/System.Management.Automation/utils/PsUtils.cs

```diff
@@ -135,41 +133,6 @@ internal static Process GetParentProcess(Process current)
             }
         }
 
-        /// <summary>
-        /// Returns processor architecture for the current process.
-        /// If powershell is running inside Wow64, then <see cref="ProcessorArchitecture.X86"/> is returned.
-        /// </summary>
-        /// <returns>Processor architecture for the current process.</returns>
-        internal static ProcessorArchitecture GetProcessorArchitecture(out bool isRunningOnArm)
-        {
-            var sysInfo = new NativeMethods.SYSTEM_INFO();
-            NativeMethods.GetSystemInfo(ref sysInfo);
-            ProcessorArchitecture result;
-            isRunningOnArm = false;
-            switch (sysInfo.wProcessorArchitecture)
-            {
-                case NativeMethods.PROCESSOR_ARCHITECTURE_IA64:
-                    result = ProcessorArchitecture.IA64;
-                    break;
-                case NativeMethods.PROCESSOR_ARCHITECTURE_AMD64:
-                    result = ProcessorArchitecture.Amd64;
-                    break;
-                case NativeMethods.PROCESSOR_ARCHITECTURE_INTEL:
-                    result = ProcessorArchitecture.X86;
-                    break;
-                case NativeMethods.PROCESSOR_ARCHITECTURE_ARM:
-                    result = ProcessorArchitecture.None;
-                    isRunningOnArm = true;
-                    break;
```

Comment:
Can you please clean up the native method, struct and const values that are no longer needed?

Reply:
Done.